lib/cocina/models/license.rb

```diff
@@ -2,9 +2,6 @@
 
 module Cocina
   module Models
-    # This vocabulary defines some of the supported licenses
-    class License < Vocabulary('https://cocina.sul.stanford.edu/licenses/')
-      property :none
-    end
```

Comment:
Jettisons the former one-off vocabulary for licenses and uses the enum provided in the API spec.